internal/authctx/client.go

```diff
@@ -26,9 +28,11 @@ const (
 )
 
 type TanzuContext struct {
+	SelfManaged      bool
 	ServerEndpoint   string
-	Token            string
-	VMWCloudEndPoint string
+	SMUsername       string
+	Token            string // selfmanaged password is stored here
```

Comment:
Reusing the Token and VMWCloudEndPoint for TMC saas and local might be bit confusing, instead can you rename them so that it will be generic?